core/src/replay_stage.rs

```diff
@@ -654,7 +654,7 @@ impl ReplayStage {
                 )
             };
             // Thread pool to (maybe) replay multiple threads in parallel
-            let replay_mode = if replay_slots_concurrently {
```

Comment:
Can we switch the contents of the `if/else` cases to avoid the `!`.

Reply:
Anything for you

Reply:
Just trying to help bump up your line counts 😉